includes/deprecated-functions.php

```diff
@@ -1157,3 +1157,87 @@ function edd_record_status_change( $payment_id, $new_status, $old_status ) {
 
 	edd_insert_payment_note( $payment_id, $status_change );
 }
+
+/**
+ * Fires the edd_before_purchase_history hook in the purchase history, if needed.
+ *
+ * @deprecated 3.0
+ * @param array $orders The array of the current user's orders.
```

Comment:
Looking at PR #8556, do we want an `@todo Formally deprecate in EDD 3.1` for these hooks, too?